src/xrpld/ledger/View.h

```diff
@@ -415,16 +446,33 @@ rippleCredit(
     ApplyView& view,
     AccountID const& uSenderID,
     AccountID const& uReceiverID,
-    const STAmount& saAmount,
+    STAmount const& saAmount,
     bool bCheckIssuer,
     beast::Journal j);
 
+[[nodiscard]] TER
+rippleCreditMPT(
```

Comment:
Unlike the others, which accept `Issue` vs `MPTIssue`, there is no type difference in the parameter list to distinguish these "overloads". What do you think of renaming the original function and all existing callers `rippleCreditIOU`? Could then add a function `rippleCredit` that takes an `STAmount` and works for any asset by delegating to the other overloads. Similar for `accountSend`.